exercises/Python_Principles.ipynb

```diff
@@ -348,6 +295,62 @@
     "Don't worry too much about the semantic differences between data types and data structures too much at this stage, but its useful to have an awareness of the term."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c626af20",
+   "metadata": {},
+   "source": [
+    "#### NB Maybe we can rephrase to make it a bit clearer? Something like: ____"
```

Comment:
@GuzalBulatova I think your explanation below is a good one. Do you want to remove the comment #### NB ?